$MirOS: ports/x11/gnome/librsvg/patches/patch-Makefile_am,v 1.1 2007/07/10 19:18:49 bsiegert Exp $
--- Makefile.am.orig	Thu Aug 31 16:47:23 2006
+++ Makefile.am	Mon Aug 17 10:49:43 2009
@@ -4,7 +4,7 @@ else
 target_rsvg_view =
 endif
 
-SUBDIRS = . moz-plugin gdk-pixbuf-loader gtk-engine data doc
+SUBDIRS = . moz-plugin gdk-pixbuf-loader gtk-engine data
 
 lib_LTLIBRARIES = librsvg-2.la
 bin_PROGRAMS = rsvg-convert $(target_rsvg_view)
@@ -70,7 +70,7 @@ librsvg_2_la_SOURCES = 		\
 	rsvg-file-util.c
 
 librsvg_2_la_LDFLAGS = -version-info @VERSION_INFO@ -export-dynamic -no-undefined -export-symbols $(srcdir)/librsvg.def
-librsvg_2_la_LIBADD = $(LIBGNOME_VFS_LIBS) $(LIBGSF_LIBS) $(LIBCROCO_LIBS) $(LIBRSVG_LIBS) $(FREETYPE_LIBS)
+librsvg_2_la_LIBADD = $(LIBGNOME_VFS_LIBS) $(LIBGSF_LIBS) $(LIBCROCO_LIBS) $(LIBRSVG_LIBS)
 
 librsvgincdir = $(includedir)/librsvg-2/librsvg
 librsvginc_HEADERS = 	\
